Automattic/newspack-plugin
# Update CPT-like Wizard headers (Campaigns and Segments)

### All Submissions:






### Changes proposed in this Pull Request:

This PR does a few things:

* Introduces a new `headerActions` prop to a `Card`
* Uses it for Campaigns and Segments

### How to test the changes in this Pull Request:

1. Navigate to Campaigns
2. Switch to this branch
3. Refresh -- does it look similar?
4. Switch back to master and navigate to Segments
5. Switch to this branch
6. Refresh -- it should look slightly different than before but more balanced with Campaigns

### Other information:
